packages/onenote-converter/build.js

```diff
@@ -0,0 +1,8 @@
+const { execSync } = require('child_process');
+
+const build = () => {
+	const profile = process.env.NODE_ENV === 'production' ? '--release' : '--debug';
+	return execSync(`wasm-pack build --target nodejs ${profile}`);
```

Comment:
Please don't use `execSync` - we should generally only use async methods so that multiple methods can run in parallel.

You might want to use `execCommand2` from the utils package instead since it will better handle errors and output

Reply:
Actually ignore this - in this context it's fine to reduce dependencies to other packages and use built-in Node functions instead